.github/workflows/ci-restart-failed-daily-conda-builds.yml

```diff
@@ -0,0 +1,33 @@
+name: Restart failed daily conda builds
+
+on:
+  workflow_run:
+    workflows: [ci-ifcopenshell-conda-daily]
+    types:
+      - completed
+
+env:
+  REPO_OWNER: IfcOpenShell
+  MY_WORKFLOW: ci-ifcopenshell-conda-daily
+
+jobs:
+  restart-failed-runs:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+    - name: Check for failed runs
+      id: check-failed-runs
+      run: |
+        echo "Checking for failed runs..."
+        # Replace "my-workflow" with the name of your workflow
+        failed_runs=$(curl -s https://api.github.com/repos/${{ env.REPO_OWNER }}/REPO/actions/runs?status=completed&conclusion=failure&workflow_id=${{ env.MY_WORKFLOW }} | jq '. | map(select(.status.code == 143))')
```

Comment:
Is `/REPO` correct here?

Reply:
I think so. It at least mangaes to find the failed jobs on my [test repo](https://github.com/Krande/condapackaging/actions/workflows/ci-restart-failed.yml) for conda packaging repo